Header header
nav_msgs/OccupancyGrid[] array
